main/src/ca/uwaterloo/flix/language/phase/Parser.scala

```diff
@@ -1089,6 +1089,10 @@ class Parser(val source: Source) extends org.parboiled2.Parser {
       SP ~ keyword("force") ~ WS ~ RecordSelect ~ SP ~> ParsedAst.Expression.Force
     }
 
+    def JavaApply: Rule1[ParsedAst.Expression] = rule {
+      SP ~ "." ~ LowerCaseVariable ~ "." ~ Names.JavaMethod ~ ArgumentList ~ SP ~> ParsedAst.Expression.JavaApply
+    }
```

Comment:
@herluf-ba Can you help Chanattan by sketching how this should look in Parser2/Weeder2?

Reply:
Strictly speaking, we should not be using ArgumentList here.

Reply:
Ok that is fine